docs/modules/discovery.md

```diff
@@ -203,6 +203,13 @@ The following list of module parametrization settings can be provided either as
                                       Type: boolean
                                       Default: true
 
+            RejectUnknownRevocationStatus=VALUE
+                                      OPC UA Client Security Config - aReject chain validation 
+                                      with CA certs with unknown revocation status, e.g.when the
```

Comment:
There should be a space between `e.g.` and `when` in `e.g.when`.

Reply:
done